packages/l10n/src/country-information.ts

```diff
@@ -34,7 +34,7 @@ export const countriesShape = PropTypes.objectOf(PropTypes.string);
  * If running through webpack, code splitting makes `getCountriesForLocale`
  * a function that asynchronously loads the country data.
  */
-const getCountriesForLocale = (locale, cb) => {
+const getCountriesForLocale = (locale: string, cb: Function) => {
```

Comment:
We should specify the callback function signature

```suggestion
const getCountriesForLocale = (locale: string, cb: (error?: Error, countries: Record<string, string>) => void) => {
```

Reply:
...or whatever the record type is (I just put string,string as an example)

Reply:
I'm not sure this works? 

https://stackoverflow.com/questions/17186566/how-do-i-fix-error-ts1015-parameter-cannot-have-question-mark-and-initializer

Reply:
The second arg should also be optional I suppose